### PR TITLE
ArQmA V0.2.0

### PR DESCRIPTION
**Pull request by ArqTras**
For review
Know bug to correct +x for contrib/depends/config files

### CODE REVIEW

**Comment by ArqTras** [comment]
Executable flags

File: Dockerfile [hunk 1]
```diff
@@ -143,5 +181,5 @@ VOLUME /wallet
 EXPOSE 19993
 EXPOSE 19994
 
-ENTRYPOINT ["arqmad", "--p2p-bind-ip=0.0.0.0", "--p2p-bind-port=19993", "--rpc-bind-ip=0.0.0.0", "--rpc-bind-port=19994", "--non-interactive", "--confirm-external-bind"]
+ENTRYPOINT ["monerod", "--p2p-bind-ip=0.0.0.0", "--p2p-bind-port=19993", "--rpc-bind-ip=0.0.0.0", "--rpc-bind-port=19994", "--non-interactive", "--confirm-external-bind"]
```
Comment (malbit) [inline Dockerfile]: That should stay at “arqmad” and should not be changed to “monerod”

File: Makefile [hunk 2]
```diff
@@ -29,117 +28,132 @@
...
 
 release-static-android:
-	mkdir -p build/release/translations
-	cd build/release/translations && cmake ../../../translations && $(MAKE)
-	cd build/release && CC=arm-linux-androideabi-clang CXX=arm-linux-androideabi-clang++ cmake -D BUILD_TESTS=OFF -D ARCH="armv7-a" -D STATIC=ON -D BUILD_64=OFF -D CMAKE_BUILD_TYPE=release -D ANDROID=true -D INSTALL_VENDORED_LIBUNBOUND=ON -D BUILD_TAG="android" -D CMAKE_SYSTEM_NAME="Android" -D CMAKE_ANDROID_STANDALONE_TOOLCHAIN="${ANDROID_STANDALONE_TOOLCHAIN_PATH}" -D CMAKE_ANDROID_ARM_MODE=ON -D CMAKE_ANDROID_ARCH_ABI="armeabi-v7a" ../.. && $(MAKE)
-
-release-static-android-gui:
-	mkdir -p build/release
-	cd build/release && CC=arm-linux-androideabi-clang CXX=arm-linux-androideabi-clang++ cmake -D BUILD_TESTS=OFF -D ARCH="armv7-a" -D STATIC=ON -D BUILD_64=OFF -D CMAKE_BUILD_TYPE=release -D ANDROID=true -D BUILD_GUI_DEPS=ON -D INSTALL_VENDORED_LIBUNBOUND=ON -D BUILD_TAG="android" -D CMAKE_SYSTEM_NAME="Android" -D CMAKE_ANDROID_STANDALONE_TOOLCHAIN="${ANDROID_STANDALONE_TOOLCHAIN_PATH}" -D CMAKE_ANDROID_ARM_MODE=ON -D CMAKE_ANDROID_ARCH_ABI="armeabi-v7a" ../.. && $(MAKE)
+	mkdir -p $(builddir)/release/translations
```
Comment (malbit) [inline Makefile]: It should be: release-static-android-armv7 instead and its release-static-android-armv8 missing

File: Makefile [hunk 3]
```diff
@@ -29,117 +28,132 @@
...
-	mkdir -p build/fuzz
-	cd build/fuzz && cmake -D STATIC=ON -D SANITIZE=ON -D BUILD_TESTS=ON -D USE_LTO=OFF -D CMAKE_C_COMPILER=afl-gcc -D CMAKE_CXX_COMPILER=afl-g++ -D ARCH="x86-64" -D CMAKE_BUILD_TYPE=fuzz -D BUILD_TAG="linux-x64" ../.. && $(MAKE)
+#fuzz:
+#	mkdir -p $(builddir)/fuzz
+#	cd $(builddir)/fuzz && cmake -D STATIC=ON -D SANITIZE=ON -D BUILD_TESTS=ON -D USE_LTO=OFF -D CMAKE_C_COMPILER=afl-gcc -D CMAKE_CXX_COMPILER=afl-g++ -D ARCH="x86-64" -D CMAKE_BUILD_TYPE=fuzz -D BUILD_TAG="linux-x64" $(topdir) && $(MAKE)
 
 clean:
 	@echo "WARNING: Back-up your wallet if it exists within ./build!" ; \
-        read -r -p "This will destroy the build directory, continue (y/N)?: " CONTINUE; \
+        read -r -p "This will destroy all build directories, continue (y/N)?: " CONTINUE; \
```
Comment (malbit) [inline Makefile]: “clean all:” should be added after and ./build folder is deprecated for repo-depend build system

File: utils/build_scripts/android32.Dockerfile [hunk 4]
```diff
@@ -139,4 +139,4 @@ RUN cd /src \
        CMAKE_LIBRARY_PATH="${PREFIX}/lib" \
        ANDROID_STANDALONE_TOOLCHAIN_PATH=${TOOLCHAIN_DIR} \
        USE_SINGLE_BUILDDIR=1 \
-       PATH=${HOST_PATH} make release-static-android -j${NPROC}
+       PATH=${HOST_PATH} make release-static-android-armv7 -j${NPROC}
```
Comment (malbit) [inline utils/build_scripts/android32.Dockerfile:142]: Over here you did add -armv7 to build but Makefile has missing it.
And where is android64.Dockerfile ??

**Review by malbit — CHANGES_REQUESTED** [review]

**Review by malbit — APPROVED** [review]
I made changes required so lets do it

**Pull request merged**